release_notes/ocp-4-15-release-notes.adoc

```diff
@@ -375,6 +375,57 @@ Pods created using the macvlan CNI plugin, where the IP address management CNI p
+
+[id="ocp-4-15-storage-gcp-filestore-shared-vpc"]
+==== Shared VPC is supported for the GCP Filestore CSI Driver Operator as generally available
+Shared virtual private cloud (VPC) for the Google Compute Platform (GCP) Container Storage Interface (CSI) Driver Operator is now supported as a generally available feature. Shared VPC simplifies network management, allows consistent network policies, and provides a centralized view of network resources.
+
+For more information, see xref:..//storage/container_storage_interface/persistent-storage-csi-google-cloud-file.adoc#persistent-storage-csi-google-cloud-file-create-sc_persistent-storage-csi-google-cloud-file[Creating a storage class for GCP Filestore Storage].
+
+[id="ocp-4-15-storage-ibm-vpc-byok"]
+==== User-Managed encryption supports IBM VPC Block storage as generally available
+The user-managed encryption feature allows you to provide keys during installation that encrypt {product-title} node root volumes, and enables all managed storage classes to use the specified encryption key to encrypt provisioned storage volumes. This feature was introduced in {product-title} 4.13 for Google Cloud Platform (GCP) persistent disk (PD) storage, Microsoft Azure Disk, and Amazon Web Services (AWS) Elastic Block storage (EBS), and is now supported on IBM Virtual Private Cloud (VPC) Block storage.
+
+//TODO: For more information, see xref:..//storage/container_storage_interface/persistent-storage-csi-ibm-vpc-block.adoc#byok_persistent-storage-csi-ibm-vpc-block [User-managed encryption].
+
+[id="ocp-4-15-storage-selinux-relabling-mount-options"]
+==== SELinux relabeling using mount options (Technology Preview)
+Previously, when SELinux was enabled, the persistent volume's (PV's) files were relabeled when attaching the PV to the pod, potentially causing timeouts when the PVs contained a lot of files, as well as overloading the storage backend.
+
+In {product-title} 4.15, for Container Storage Interface (CSI) driver that support this feature, the driver will mount the volume directly with the correct SELinux labels, eliminating the need to recursively relabel the volume, and pod startup can be significantly faster.
+
+This feature is supported with Technology Preview status. 
+
+If the following conditions are true, the feature is enabled by default:
+
+* The CSI driver that provides the volume has support for this feature with `seLinuxMountSupported: true` in its CSIDriver instance. The following CSI drivers that are shipped as part of {product-title} announce SELinux mount support:
+
+** AWS Elastic Block Storage (EBS)
+** Azure Disk
+** Google Compute Platform (GCP) persistent disk (PD)
+** IBM Virtual Private Cloud (VPC) Block
+** OpenStack Cinder
+** VMware vSphere
+
+* Ensure the pod that uses the persistent volume has full SELinux label specified in its `spec.securityContext` or `spec.containers[*].securityContext` by using `restricted` SCC and set access  mode `ReadWriteOncePod` for the volume.
```

Comment:
I think we should make it more obvious that it works ONLY with RWOP, the others access modes still rely on the old method.

Reply:
Called it out in separate bullet point. Fixed.